google/cloud/aiplatform/utils/worker_spec_utils.py

```diff
@@ -21,6 +21,13 @@
     accelerator_type as gca_accelerator_type_compat,
 )
 
+SPEC_ORDERS = {
```

Comment:
```suggestion
_SPEC_ORDERS = {
```
Also add a comment above about the purpose of this mapping.